src/lib/openapi/spec/personal-dashboard-schema.ts

```diff
@@ -7,6 +7,41 @@ export const personalDashboardSchema = {
     additionalProperties: false,
     required: ['projects', 'flags'],
     properties: {
+        admins: {
```

Comment:
do we want to make admins and project owners required?

Reply:
We *can*, but I didn't want to do it yet, in case we decide to only return those when the user has no projects.